src/lib/receipt_chain_database_lib/dune

```diff
@@ -0,0 +1,9 @@
+(library
+ (name receipt_chain_database_lib)
```

Comment:
instead of calling it `receipt_chain_database_lib` how about `receipt_chain_database`?